### PR TITLE
sci-physics/fastjet-contrib: Fix tc ar and ranlib

### PR DESCRIPTION
**Pull request by APN-Pucky**
Closes: https://bugs.gentoo.org/860222
Signed-off-by: Alexander Puck Neuwirth <alexander@neuwirth-informatik.de>

### CODE REVIEW

**Comment by amadio** [comment]
Isn't this the same as #26554, and #26620? The change seems fine, but I don't get why there are several PRs with the same change being opened. You can rebase and push to the same PR if you update/rebase it.

**Comment by APN-Pucky** [comment]
Sorry for the mess. 
I thought adding my self to the proxy maintainers would already add the label, but I guess it only gets added once the current metadata.xml has the proxy-maintainer.
In the other one I forgot to add the Bug to be closed.

**Comment by amadio** [comment]
No problem, I'm just trying to understand why. Adding yourself as proxy-maintainer should be a separate commit, though. Do you mind splitting it and force-pushing here?

File: sci-physics/fastjet-contrib/files/fastjet-contrib-1.049-ar.patch [hunk 1]
```diff
@@ -0,0 +1,266 @@
+diff -Naru fjcontrib-1.049.orig/Centauro/Makefile fjcontrib-1.049/Centauro/Makefile
```
Comment (SoapGentoo) [inline sci-physics/fastjet-contrib/files/fastjet-contrib-1.049-ar.patch:1]:
```suggestion
```

File: sci-physics/fastjet-contrib/files/fastjet-contrib-1.049-ar.patch [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+--- fjcontrib-1.049.orig/Centauro/Makefile	2022-07-24 00:55:40.912538473 +0200
```
Comment (SoapGentoo) [inline sci-physics/fastjet-contrib/files/fastjet-contrib-1.049-ar.patch:2]:
```suggestion
--- a/Centauro/Makefile
```

File: sci-physics/fastjet-contrib/files/fastjet-contrib-1.049-ar.patch [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
++++ fjcontrib-1.049/Centauro/Makefile	2022-07-24 01:20:00.488103968 +0200
```
Comment (SoapGentoo) [inline sci-physics/fastjet-contrib/files/fastjet-contrib-1.049-ar.patch:3]:
```suggestion
+++ b/Centauro/Makefile
```

**Review by SoapGentoo — CHANGES_REQUESTED** [review]
please make patches as lean and "git alike" as possible, the metadata is just noise

File: sci-physics/fastjet-contrib/metadata.xml [hunk 4]
```diff
@@ -1,6 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
+	<maintainer type="person" proxied="yes">
+		<email>alexander@neuwirth-informatik.de</email>
+		<name>Alexander Puck Neuwirth</name>
+	</maintainer>
+	<maintainer type="project" proxied="proxy">
```
Comment (AndrewAmmerlaan) [inline sci-physics/fastjet-contrib/metadata.xml:8]: CC @gentoo/proxy-maint for new proxy maintainer

**Review by AndrewAmmerlaan — APPROVED** [review]

File: sci-physics/fastjet-contrib/fastjet-contrib-1.049.ebuild [hunk 5]
```diff
@@ -22,12 +22,14 @@ RDEPEND="${DEPEND}"
...
 PATCHES=(
 	"${FILESDIR}"/${P}-soname.patch
+	"${FILESDIR}"/${P}-ar.patch
 )
 
 src_configure() {
-	CXX=`tc-getCXX`
-	./configure --prefix=/usr --fastjet-config=/usr/bin/fastjet-config CXX="${CXX}" CFLAGS="${CFLAGS}" CXXFLAGS="${CXXFLAGS}" FFLAGS="${FFLAGS}" LDFLAGS="${LDFLAGS}" || die
+	tc-export CXX AR RANLIB
+	./configure --prefix=/usr --fastjet-config=/usr/bin/fastjet-config RANLIB="${RANLIB}" AR="${AR}" CXX="${CXX}" CFLAGS="${CFLAGS}" CXXFLAGS="${CXXFLAGS}" FFLAGS="${FFLAGS}" LDFLAGS="${LDFLAGS}" || die
```
Comment (AndrewAmmerlaan) [inline sci-physics/fastjet-contrib/fastjet-contrib-1.049.ebuild:30]: I think it is a good idea to revision bump the ebuild to -r1. I'll do this while merging.

**Comment by AndrewAmmerlaan** [comment]
Thanks :+1:

**Pull request closed without merging**